packages/@uppy/companion/src/server/controllers/url.js

```diff
@@ -71,8 +72,7 @@ const get = (req, res) => {
       res.status(response.status).json(response.body)
     }).catch((err) => {
       logger.error(err, 'controller.url.get.error', req.id)
-      // @todo this should send back error (not err)
-      res.json({ err })
+      return res.status(err.status || 500).json({ message: 'failed to fetch URL metadata' })
```

Comment:
is this OK for old clients or do we need to check the uppy version?

Reply:
the old clients were never able to read this message because of this wrong format, so they default to constructing a message from the status code instead. See [here](https://github.com/transloadit/uppy/blob/master/packages/%40uppy/companion-client/src/RequestClient.js#L86)

Reply:
from [older versions](https://github.com/transloadit/uppy/blob/uppy%401.0.0/packages/@uppy/companion-client/src/RequestClient.js#L73)

Reply:
ok perfect, thanks!